### ISSUE TITLE
You seem to have deleted the migrations rake task?

### ISSUE THREAD

**Issue by kulte** [instacart/amountable#10]
😕 There are no rake tasks in this gem 😄

**Comment by kulte** [comment]
I'm sorry, but how did this get messed up so badly 😆 The current state of this repo is really horrible, right?

**Comment by Tressa-Sanders** [comment]
Yep, this makes no sense. There's a whole article about this gem only to not even be able to install it correctly when you get here.
